### PR TITLE
fix: <a> cannot appear as descendant of <a>

### PR DESCRIPTION
**Pull request by gikf**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

---
- Fixes console warning

### CODE REVIEW

File: client/src/modules/events/pages/eventPage.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import {
   Heading,
   HStack,
   Image,
+  Link as ChakraLink,
```
Comment (Sboonny) [inline client/src/modules/events/pages/eventPage.tsx:9]:
```suggestion
  Link,
```

File: client/src/modules/events/pages/eventPage.tsx [hunk 2]
```diff
@@ -381,9 +382,9 @@ export const EventPage: NextPage = () => {
         {data.event.url && (
           <Text fontWeight={'500'} fontSize={['smaller', 'sm', 'md']}>
             More about event:{' '}
-            <Link href={data.event.url} isExternal>
+            <ChakraLink href={data.event.url} isExternal>
```
Comment (Sboonny) [inline client/src/modules/events/pages/eventPage.tsx:385]:
```suggestion
            <Link href={data.event.url} isExternal>
```

File: client/src/modules/events/pages/eventPage.tsx [hunk 3]
```diff
@@ -381,9 +382,9 @@ export const EventPage: NextPage = () => {
...
               {data.event.url}
-            </Link>
+            </ChakraLink>
```
Comment (Sboonny) [inline client/src/modules/events/pages/eventPage.tsx:387]:
```suggestion
            </Link>
```

**Review by Sboonny — COMMENTED** [review]
What do you think of deleting `import { Link } from 'chakra-next-link';`, it is getting in the way and doesn't provide much more than the chakra `Link`, in this file.

**Comment by gikf** [comment]
@Sboonny it still allows to not load chapter page every time when navigating to it via link on the event page. Try it:
1. Go to `/chatpers/1`.
2. Navigate to first event.
3. Navigate to chapter page via link at the event page.

With chakra's `Link` step 3 will make a fresh load every time. With chakra-next-link's `Link` cache is used.

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Pull request merged**